packages/manager/modules/pci/src/projects/project/instances/instance/application-access/application-access.html

```diff
@@ -0,0 +1,46 @@
+<oui-modal
+    data-heading="{{ 'pci_projects_project_instances_instance_application-access_title' | translate }}"
+    data-on-dismiss="$ctrl.goBack()">
+
+    <div data-ng-repeat="access in $ctrl.applicationAccesses.accesses track by access.type"
+        data-ng-class="{
+            'border-bottom': !$last
+        }" class="py-3 container-fluid">
+
+        <div class="row pb-2" data-ng-if="access.type">
```

Comment:
Do you think we could iterate on `access` and display the associated information ? Does it make sense ?

Reply:
Nope, `access` contains only `type, url, login, password`